### PR TITLE
General cleanup items

### PR DESCRIPTION
**Pull request by jkirk5**
### Summary

- Replaced use of "static" and "dynamic" with "pre-mission" and "mission". NOTE: onboarding_level1.ipynb needs new screenshots to reflect latest structure of Aviary problems
- Renamed references to GASP turboprop decks to turboshaft decks for better accuracy. GASP_TP Enum changed to GASP_TS (used for engine deck conversion util)

Usage of verbosity was overhauled:
- Verbosity is now IntEnum to allow for direct equality/inequality comparisons
- All uses of ".value" for Verbosity have been removed - this should not get used in the future, as it prevents users from using ints to define verbosity during function calls
- For examples/tests/places where verbosity is used for calling functions, use ints for shorter arguments and fewer imports
- For function/class definitions/source code, use verbosity Enum for readability
- Always use comparators such as "==" and "<" for checking Verbosity level, for compatibility with users passing ints

### Related Issues

None

### Backwards incompatibilities

None

### New Dependencies

None

### CODE REVIEW

**Comment by xjjiang** [comment]
For verbosity, I see we have both `verbosity=0` and `verbosity=Verbosity.QUIET`. Should we be more consistent?

**Comment by xjjiang** [comment]
What is OpenMDAO system2?

**Comment by jkirk5** [comment]
> For verbosity, I see we have both `verbosity=0` and `verbosity=Verbosity.QUIET`. Should we be more consistent?

Your comment didn't get attached to a file - if it is using both within the same file I'd like to be consistent. 

Using either method works and is intended and supported. I'm using the following convention:
- For examples/tests/places where verbosity is used for calling functions, use ints for shorter arguments and fewer imports
- For function/class definitions/source code, use verbosity Enum for readability

Basically, in source code itself I'm using the Enum, and when it is a script or just calling the function, I'm using the ints to avoid needing to import the enum and making the function call shorter (mostly for tests, which are more developer-focused)

**Comment by jkirk5** [comment]
> What is OpenMDAO system2?

Just a typo, thanks for catching!!

**Comment by xjjiang** [comment]
> > For verbosity, I see we have both `verbosity=0` and `verbosity=Verbosity.QUIET`. Should we be more consistent?
> 
> Your comment didn't get attached to a file - if I used both within the same file I'd like to be consistent.
> 
> Using either method works and is intended and supported. I'm using the following convention:
> 
>     * For examples/tests/places where verbosity is used for calling functions, use ints for shorter arguments and fewer imports
> 
>     * For function/class definitions/source code, use verbosity Enum for readability
> 
> 
> Basically, in source code itself I'm using the Enum, and when it is a script or just calling the function, I'm using the ints to avoid needing to import the enum and making the function call shorter (mostly for tests, which are more developer-focused)

Got it. Will you write about this convention in docs (e.g., `user_guide/troubleshooting.ipynb`)?

**Review by Kenneth-T-Moore — APPROVED** [review]

**Comment by jkirk5** [comment]
> Got it. Will you write about this convention in docs (e.g., `user_guide/troubleshooting.ipynb`)?

Yeah that's a good idea. I'll look around for the best place to mention this convention in the docs.

File: aviary/docs/user_guide/aviary_commands.ipynb [hunk 1]
```diff
@@ -274,16 +274,16 @@
...
+    "- Turboshaft decks for GASP can also be converted\n",
     "- Comments can be added using #\n",
     "\n",
     "\n",
     "Example usage:\n",
     "```\n",
     "`aviary convert_engine turbofan_23k_1.eng turbofan_23k_1_lbm_s.deck -f GASP` Convert a GASP based turbofan\n",
     "`aviary convert_engine -f FLOPS turbofan_22k.eng turbofan_22k.txt` Convert a FLOPS based turbofan\n",
-    "`aviary convert_engine turboprop_4465hp.eng turboprop_4465hp.deck --data_format GASP_TP` Convert a GASP based turboprop\n",
+    "`aviary convert_engine turbohaft_4465hp.eng turboshaft_4465hp.deck --data_format GASP_TS` Convert a GASP based turboshaft\n",
```
Comment (ehariton) [inline aviary/docs/user_guide/aviary_commands.ipynb:286]: turbohaft_4465hp missing and 's'

File: aviary/interface/test/test_basic_report.py
Comment (ehariton) [inline aviary/interface/test/test_basic_report.py:1]: Doubt we need this file here?

File: aviary/mission/gasp_based/phases/time_integration_traj.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import numpy as np
 from aviary.mission.gasp_based.ode.time_integration_base_classes import SGMTrajBase
 from aviary.mission.gasp_based.phases.time_integration_phases import SGMGroundroll, SGMRotation
+from aviary.variable_info.enums import Verbosity
```
Comment (ehariton) [inline aviary/mission/gasp_based/phases/time_integration_traj.py:4]: This is the first example I've seen so far of where you've added Verbosity back instead of using a numerical assignment like what you did in test_linkage_logic.py. Was this on purpose or was this an oversight?

**Review by ehariton — APPROVED** [review]

File: aviary/interface/test/test_basic_report.py
Comment (jkirk5) [reply aviary/interface/test/test_basic_report.py:1]: removed
